GANA-UI.package/GanaBranchGraph.class/instance/createGraph.st

```diff
@@ -0,0 +1,10 @@
+algorithm
+createGraph
+
+	self initializeAlgorithm.
+	commitList withIndexDo: [ :commit :index |
+		self addEdgesToChildrenFrom: commit.
```

Comment:
I don't really like the wording here since we add _ourselves_ as children to our parents, not the other way round. The edges to our children get painted in the next step... What did you try to express with it @feeds?